lib/bindings/AMQPBinding.js

```diff
@@ -86,6 +86,14 @@ function start(callback) {
         queue = constants.AMQP_DEFAULT_QUEUE;
     }
 
+    var durable;
+
+    if (config.getConfig() && config.getConfig().amqp && config.getConfig().amqp.durable) {
+        durable = config.getConfig().amqp.durable;
+    } else {
+        durable = constants.AMQP_DEFAULT_DURABLE;
+    }
```

Comment:
Looking how durable is being processed here I guess it should be (inside config.amqp):

```
    durable: true,
```

However, in config.js I see (https://github.com/telefonicaid/iotagent-json/blob/master/config.js#L100):

```
    options: { durable: true }
```

Maybe config.js should be fixed?

Reply:
fixed in https://github.com/telefonicaid/iotagent-json/pull/448/commits/26774c007854239ca7bdb953dace103c10c4c714